### PR TITLE
Reports Overview

### PR DESCRIPTION
**Pull request by shahswet**

### CODE REVIEW

File: pages/planetusermanual.md [hunk 1]
```diff
@@ -236,3 +236,40 @@ Notes:
 4. **Open Members Detail** - everyone can see the details of a member here.
 5. **Edit Members Detail** - this is an admin only functionality for editing a members account information.
 6. **Delete a Member** - this is an admin only functionality for deleting members.
+
+# Report Overview
+
+## _What is a Report?_
```
Comment (kylemathias) [inline pages/planetusermanual.md]: make sure to make all the formating uniform to all the other headers on the page.
change ```## _What is a Report?_``` to ```### _What is a Report?_```

File: pages/planetusermanual.md [hunk 2]
```diff
@@ -236,3 +236,40 @@ Notes:
...
+Reports allows distribution of Nation Reports as needed by Nation Managers. Managers can add new Reports, check the activity on each Report, and look at trends for each Report.
+
+## _How do I create a Report?_
```
Comment (kylemathias) [inline pages/planetusermanual.md]: same here ```## _How do I create a Report?_``` to ```### _How do I create a Report?_```

**Comment by shahswet** [comment]
@Supriyaranjan @kylemathias Is it okay now?

**Comment by Supriyaranjan** [comment]
@shahswet Did you added "Members menu"? I am sorry but I didn't find that section in the original document. Can you tell me about it?

**Comment by shahswet** [comment]
@Supriyaranjan I didn't add that Members menu. It was there previously. I started from Reports Overview.

**Comment by shahswet** [comment]
@Supriyaranjan @kylemathias @dogi I have made the changes. Can you review it?

**Pull request merged**